akka-docs/src/test/java/jdocs/stream/operators/SourceOrFlow.java

```diff
@@ -327,4 +327,26 @@ void takeWhileExample() {
     // 2
     // #take-while
   }
+
+  Source<String, NotUsed> someText() {
+    return Source.from(
+        Arrays.asList(
+            ("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt "
+                    + "ut labore et dolore magna aliqua.")
+                .split(" ")));
+  }
+
+  void filterExample() {
+    // #filter
+    Source<String, NotUsed> words = someText();
+    Source<String, NotUsed> longWords = words.filter(w -> w.length() > 6);
+    // #filter
+  }
+
+  void filterNotExample() {
+    // #filterNot
+    Source<String, NotUsed> words = someText();
+    Source<String, NotUsed> longWords = words.filter(w -> w.length() <= 5);
+    // #filterNot
+  }
```

Comment:
I'd include what the source so it shows what it will emit  and also add a `// prints` (even if that may seem obvious)

Reply:
ok, I have added the full source and sink